fg/settings/config.py

```diff
@@ -7,18 +7,45 @@
 with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 '''
+import os
+from scif.logger import bot
+
+def getenv(variable_key, default=None, required=False, silent=True, boolean=False):
+    ''' getenv will attempt to get an environmental variable. If the variable
+        is not found, the deafult value will be returned.
+
+        Parameters
+        ==========
+        variable_key: the variable name to be searched for
+        required: exit with error if not found
+        silent: if silent, does not print debugging information
+
+        All credit to https://github.com/vsoch/scif/blob/master/scif/defaults.py
+    '''
+    variable = os.environ.get(variable_key)
+    if variable is None and required == True:
+        bot.exit("Cannot find environment variable {}, exiting.".format(variable_key))
+
+    if silent != True and variable is not None:
+        bot.verbose("{} found as {}".format(variable_key,variable))
+
+    if boolean == True:
+        variable = variable == 'True' # Handles booleans
+
+    return variable
+
 
 # AUTHENTICATION
 
 # Which social auths do you want to use?
-ENABLE_GLOBUS_AUTH = True
-ENABLE_GOOGLE_AUTH = False
-ENABLE_ORCID_AUTH = False
-ENABLE_ORCID_AUTH_SANDBOX = False
-ENABLE_TWITTER_AUTH = False
-ENABLE_GITHUB_AUTH = True
-ENABLE_GITLAB_AUTH = False
-ENABLE_BITBUCKET_AUTH = False
+ENABLE_GLOBUS_AUTH = getenv('FG_ENABLE_GLOBUS_AUTH',default=False,boolean=True)
+ENABLE_GOOGLE_AUTH = getenv('FG_ENABLE_GOOGLE_AUTH',default=False,boolean=True)
+ENABLE_ORCID_AUTH = getenv('FG_ENABLE_ORCID_AUTH',default=False,boolean=True)
+ENABLE_ORCID_AUTH_SANDBOX = getenv('FG_ENABLE_ORCID_AUTH_SANDBOX',default=False,boolean=True)
+ENABLE_TWITTER_AUTH = getenv('FG_ENABLE_TWITTER_AUTH',default=False,boolean=True)
+ENABLE_GITHUB_AUTH = getenv('FG_ENABLE_GITHUB_AUTH',default=True,boolean=True)
+ENABLE_GITLAB_AUTH = getenv('FG_ENABLE_GITLAB_AUTH',default=False,boolean=True)
+ENABLE_BITBUCKET_AUTH = getenv('FG_ENABLE_BITBUCKET_AUTH',default=False,boolean=True)
 
```

Comment:
You can find the index for the development docs in https://github.com/vsoch/freegenes/blob/master/docs/_docs/development/index.md, and most of the settings are described in setup.md. But you might want to add another new file to link under deployments for what you are developing, and then just briefly mention the environment options in setup.md and link to it. Since this is jekyll, from the root of the docs (the docs folder) you should be able to run `bundle exec jekyll serve` or just `jekyll serve` to render them in your local browser.